apps/passport-server/src/database/queries/devconnect_pretix_tickets/insertDevconnectPretixTicket.ts

```diff
@@ -0,0 +1,27 @@
+import { Pool } from "pg";
+import { DevconnectPretixTicket } from "../../models";
+import { sqlQuery } from "../../sqlQuery";
+
+/**
+ * Insert a Devconnect pretix ticket into the database, if they have not been
+ * inserted yet. This does not insert an identity commitment for them.
+ */
+export async function insertDevconnectPretixTicket(
```

Comment:
may be worth changing the behavior of this function to be an upsert

Reply:
going to keep it as non-upsert for now since we already have update logic